components/UserDetails.tsx

```diff
@@ -46,11 +54,11 @@ const UserDetails = ({
         <Modal.Header closeButton>
           <Modal.Title>
             <User size={30} style={{ marginRight: 10 }} />
-            {user.email}
+            {user?.email || user.name}
```

Comment:
if user is undefined that you will get an error from `user.name`, so you should add a question mark to both.

Reply:
Name is a required field in the User type but yeah I guess better safe than sorry!

Reply:
Added in 2939ab097c940f09b81fd56be5f7e0eee0cf5220

Reply:
More about the user object being potentially undefined rather than name. If you're not ever expecting user to be null then I don't think you need a ? on either side of the || here.